install/certmonger/dogtag-ipa-ca-renew-agent-submit

```diff
@@ -135,7 +135,7 @@ def call_handler(_handler, *args, **kwargs):
         cookie = os.environ.pop('CERTMONGER_CA_COOKIE', None)
         if cookie is not None:
             try:
-                context = json.loads(cookie)
+                context = json.loads(ipautil.decode_json(cookie))
```

Comment:
This is not necessary as `cookie` is always an instance of `str` here in Python 3.